tests/mpmc_channel.rs

```diff
@@ -661,28 +712,30 @@ macro_rules! gen_mpmc_tests {
                 assert_eq!(1, elem3.strong_count());
             }
         }
-    }
+    };
 }
 
-gen_mpmc_tests!(local_mpmc_channel_tests, LocalChannel, LocalUnbufferedChannel);
+gen_mpmc_tests!(
+    local_mpmc_channel_tests,
+    LocalChannel,
+    LocalUnbufferedChannel
+);
 
 #[cfg(feature = "std")]
 mod if_std {
     use super::*;
-    use futures_intrusive::channel::{Channel, UnbufferedChannel, shared::channel,
-        shared::Sender, shared::Receiver,
-        shared::ChannelReceiveFuture, shared::ChannelSendFuture};
+    use futures_intrusive::channel::{
+        shared::channel, shared::ChannelReceiveFuture, shared::ChannelSendFuture, shared::Receiver,
+        shared::Sender, Channel, UnbufferedChannel,
+    };
 
```

Comment:
There is no test for the `Stream` retrieved from the `shared::Receiver`, right? Would be nice to have one, in order to avoid that this one gets accidentally broken.